pixi.toml

```diff
@@ -0,0 +1,94 @@
+[project]
+name = "imod-python"
+version = "0.15.1"
+description = "Make massive MODFLOW models!"
+authors = ["Deltares <huite.bootsma@deltares.nl>", ]
+channels = ["conda-forge", ]
+platforms = ["win-64", "linux-64", "osx-arm64", "osx-64"]
+license = "MIT"
+license-file = "LICENSE"
+readme = "README.rst"
+homepage = "https://github.com/Deltares/imod-python"
+documentation = "https://deltares.github.io/imod-python/"
+repository = "https://github.com/Deltares/imod-python.git"
+
+[tasks]
+docs =  { cmd = "make html", depends_on = ["install"], cwd = "docs" }
+install = "python -m pip install --no-deps --editable ."
+format = { depends_on = ["isort_format", "black_format", "ruff_format"] }
+lint = { depends_on = ["isort_lint", "black_lint", "ruff_lint"] }
+tests = { cmd = [
+    "NUMBA_DISABLE_JIT=1",
+    "pytest",
+    "--cache-clear",
+    "--verbose",
+    "--junitxml=report.xml",
+    "--cov=imod",
+    "--cov-report=term",
+    "--cov-report=html:coverage",
+    "--cov-config=.coveragerc"
+], depends_on = ["install"], cwd = "imod/tests" }
+
+black_lint = "black --check ."
+black_format = "black ."
+isort_lint = "isort --skip-gitignore --check ."
+isort_format = "isort --skip-gitignore ."
+mypy_lint = { cmd ="mypy", depends_on = ["install"]}
+mypy_report = { cmd ="mypy | mypy2junit > mypy-report.xml", depends_on = ["install"]}
+ruff_lint = "ruff check ."
+ruff_format = "ruff format ."
+
+[dependencies]
+affine = "*"
+black = "*"
+bottleneck = "*"
+cftime = ">=1"
+contextily = ">=1"
+cytoolz = "*"
+dask = "*"
+fastcore = "*"
+flopy = "*"
+geopandas = "*"
+graphviz = "*"
+hypothesis = "*"
+isort = "*"
+jinja2 = "*"
+matplotlib = "*"
+mypy = "*"
+netcdf4 = "*"
+numba = ">=0.50"
+numpy = "*"
+pandamesh = "*"
+pandas = "<2.0"  # TODO: unpin when newer xarray on conda-forge
```

Comment:
Note to ourselves: We also need to update this file when https://github.com/Deltares/imod-python/pull/715 will be merged